### PR TITLE
Add docs for installation via docker

### PR DESCRIPTION
**Pull request by hblitza**
This adds docs for installation via docker. Since there's no official docker image yet, a short example for the kartoza image was added.

### CODE REVIEW

**Comment by justb4** [comment]
My $2ct. I also provide a Docker Image, in both GitHub and DockerHub. But an officia MPl Image would be welcome.

 Been using this for years now. Originally cloned from [yagajs](https://github.com/yagajs/docker-mapproxy), Arne Schubert. That repo does not seemed to be maintained:

* https://github.com/justb4/docker-mapproxy
* https://hub.docker.com/repository/docker/justb4/mapproxy/general

Maybe this is useful. Have some other MapProxy Docker Images as well, with embedded MapServer and Mapnik.

**Comment by hblitza** [comment]
Very good, I did't mention the yagajs image because lack of maintaning - but now I had a look at your fork. I added it accordingly. What do you think @justb4 ?

**Comment by justb4** [comment]
@hblitza yes thanks. Ok like this. Maybe at some point have a dedicated Image under this GH organisation.
 I am maintaining that Image actively. Happy to receive contributions.

File: doc/install_docker.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Create a directory (e.g. `mapproxy`) for your configuration files and mount it as a volume:
+
+::
+
+  docker run --name "mapproxy" -p 8080:8080 -d -t -v `pwd`/mapproxy:/mapproxy kartoza/mapproxy
+
+Afterwards, the `MapProxy` instance is running on `localhost:8080`.
+
+I a production environment you might want to put a `nginx`_ in front of the MapProxy container, that serves as a reverse proxy.
```
Comment (weskamm) [inline doc/install_docker.rst:28]:
```suggestion
In a production environment you might want to put a `nginx`_ in front of the MapProxy container, that serves as a reverse proxy.
```

**Review by weskamm — APPROVED** [review]

**Comment by hblitza** [comment]
Thanks for review @weskamm @justb4 feel free to merge.

**Pull request merged**